alert.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="style-alert.css">
+    <title>Alerts</title>
+</head>
+<body>
+
+    <!-- alert without icon and close button -->
+    <div class="alert alert-primary">           
+        <strong>Primary</strong> this is a primary alert.
+    </div>
+
+    <!-- alert align to center -->
+    <div class="alert flex-row alert-success">        
+        <span class="flex-row">
+            <span class="material-icons mar-rg-sm">check</span>
+            <span><strong>Success</strong> your action is successful.</span>
+        </span>
+        <span class="material-icons icon">close</span>
+    </div>
+
+    <!-- alert with icon and close button -->
+    <div class="alert-container">
+        <div class="alert flex-row alert-success">        
+            <span class="flex-row">
+                <span class="material-icons mar-rg-sm">check</span>
+                <span><strong>Success</strong> your action is successful.</span>
```

Comment:
mar-rg-sm? is it saying something margin-right-small? Can improve on class names